ingress/controllers/nginx/README.md

```diff
@@ -120,7 +119,13 @@ Please follow [test.sh](https://github.com/bprashanth/Ingress/blob/master/exampl
 
 Check the [example](examples/tls/README.md)
 
+### HTTP Strict Transport Security
 
+HTTP Strict Transport Security (HSTS) is an opt-in security enhancement specified through the use of a special response header. Once a supported browser receives this header that browser will prevent any communications from being sent over HTTP to the specified domain and will instead send all communications over HTTPS.
+
+By default the controller redirects (301) to HTTPS if there is a TLS Ingress rule. 
+
+To disable this behavior use `force-hsts=false` in the NGINX ConfigMap.
```

Comment:
can we call this just hsts=true/false? since hsts is supposed to force https it feels redundant to say force hsts.